docs/UserGuide.adoc

```diff
@@ -52,126 +54,141 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
-****
+==== Registers a new patient: `register` ====
+Registers a new patient +
+Format:  `register -name|-n <PATIENT_NAME> -ic|-i <PATIENT_IC> -hp <PHONE_NUM> -email|-e <EMAIL> [-address|-a <ADDRESS>] [-remark|-r <REMARK>]`
 
-Examples:
 
-* `edit 1 p/91234567 e/johndoe@example.com` +
-Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-* `edit 2 n/Betsy Crower t/` +
-Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+==== Updates patients’ profiles: `update` ====
+If user is in the patients listing screen, updates any change for patients’ profiles +
+Format: `update <ENTRY_ID> [-name|-n <PATIENT_NAME>] [-ic|-i <PATIENT_IC>] [-hp <PHONE_NUM>] [-email|-e <EMAIL>] [-address|-a <ADDRESS>] [-remark|-r <REMARK>]`
 
-=== Locating persons by name: `find`
 
-Finds persons whose names contain any of the given keywords. +
-Format: `find KEYWORD [MORE_KEYWORDS]`
+=== On-Duty Doctors Management ===
 
-****
-* The search is case insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
-****
+==== Views doctors on shift: `doctors` ====
 
-Examples:
+Displays doctors currently on shift +
+Format: `doctors [<DOCTOR_NAME>]`
 
-* `find John` +
-Returns `john` and `John Doe`
-* `find Betsy Tim John` +
-Returns any person having names `Betsy`, `Tim`, or `John`
+==== Register a new doctor: `newdoctor` ====
+Hires a new doctor. +
+Format: `newdoctor [-name|-n <DOCTOR_NAME>] [-hp <PHONE_NUM>] +
+ [-email|-e <EMAIL>] [-remark|-r <REMARK>]`
 
-// tag::delete[]
-=== Deleting a person : `delete`
+==== Update a patient's profile: `update` ====
 
-Deletes the specified person from the address book. +
-Format: `delete INDEX`
+If user is in the doctors listing screen, updates any change for patients’ profiles. +
+Format: `update <ENTRY_ID> [-name|-n <DOCTOR_NAME>]+
+[-ic|-i <DOCTOR_IC>] [-hp <PHONE_NUM>] [-email|-e <EMAIL>] [-address|-a <ADDRESS>] +
+[-remark|-r <REMARK>]`
 
-****
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index *must be a positive integer* 1, 2, 3, ...
-****
+==== Updates patients’ profiles: `onduty` ====
+Marks the doctor as on-duty and stationed in the given room number. +
+Patients can be directed to the specified doctor. +
+Format: `onduty <ROOM_NUMBER> <DOCTOR_NAME>`
```

Comment:
please change to `onduty <ROOM_NUMBER> <DOCTOR_ENTRY_INDEX>`. Also do mention that this is only applicable when the user is on the list of doctors view.